test/minikube/001-kube.bats

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bats
+#
+# Tests of podman kube commands with minikube
+#
+
+load ../system/helpers.bash
+
+function setup(){
+    # only set up the minikube cluster before the first test
+    if [ "$BATS_TEST_NUMBER" -eq 1 ]; then
+        minikube start
```

Comment:
(I'm assuming setup() is called before every test)
Suggestion: Does minikube share state between startups?  If so, it would be helpful as a general testing strategy to give every "start" it's own unique state.  This prevents bad data from one failed test impacting the operations of the next test.

Reply:
We don't want to start a minikube cluster before every test, it takes quite a few seconds for a cluster to come up, so to delete and restart before each test will be too time consuming, especially as we add more tests. That is why, in the setup function I have a check to see if we are at the first test and to only spin up a minikube cluster then.

I can add a cleanup function that cleans up anything created during the test at the end of it so we basically have a fresh cluster for each test without having to start and delete it for every test.

Reply:
Can I do this in a follow up PR? I would like to get this in first so we can unblock the kube apply PR

I am also separating the tests out into different k8s namespaces, so one test shouldn't affect another if something goes wrong.

Reply:
Yep no problem, and I like your namespace idea too.